scripts/wasm-rebuild/docker-scripts/rebuild_tags.sh

```diff
@@ -228,6 +228,7 @@ mkdir -p "${OUTPUTDIR}"/bin
 echo "Prepare solc-js."
 cd /root/solc-js
 npm install >/dev/null 2>&1
+npm run build >/dev/null 2>&1
```

Comment:
Not sure about this one and these scripts are not actively tested but I'm adding it just in case.